src/mark.js

```diff
@@ -280,7 +280,19 @@ class Mark { // eslint-disable-line no-unused-vars
         default:
             return `()(${str})`;
         case "complementary":
-            return `()([^\\s${lsJoin}]*${str}[^\\s${lsJoin}]*)`;
+            lsJoin = lsJoin.length ?
+                `^\\s${lsJoin}` :
+                [
+                    "A-Za-z\-", // Basic Latin + dash
+                    "\u00AD", // Soft hyphen
+                    "\u00AF", // Macron
+                    "\u00B4", // Acute accent
+                    "\u00B8", // Cedilla
+                    "\u00C0-\uD7FF",
+                    "\uF900-\uFDCF",
+                    "\uFDF0-\uFFEF"
```

Comment:
Another question: Why are e.g. a macron or a cedilla characters that are necessary to capture? I mean, they aren't separate characters, or am I missing something?